.eslintrc.json

```diff
@@ -4,7 +4,7 @@
   "plugins": ["@typescript-eslint", "prettier"],
   "extends": ["standard-with-typescript", "prettier", "prettier/@typescript-eslint", "eslint-config-prettier"],
   "parserOptions": {
-    "project": "./tsconfig.json",
+    "project": "./tsconfig.eslint.json",
```

Comment:
I think this means the new file is the only one used during the build? Therefore the options are incorrect and the build might be off

Reply:
Will double check this soon.. i just want be totally sure that we end up with the right output

Reply:
I was talking rubbish - this only affects what eslint sees.